### ISSUE TITLE
Poison's Inebriation Effect and Virulent Quicksand in Yagroot Swamps leads to some unfair damage/kills to players

### ISSUE THREAD

**Issue by oitsjustjose** [TunefulTurnip/Deep-Aether#166]
Was watching Ellpeck's stream when he fell into Poison in a Yagroot Swamp. Upon getting out of the poison, he walked into Virulent Quicksand. Because poison inflicts inebriation upon the player, he was forced to move while standing in the quicksand - each movement causes suffocation damage (on top of the poison damage he had just received) and ultimately died to the biome itself. Seems pretty harsh - maybe a change to Virulent Quicksand that prevents it from applying damage if the player has Inebriation would be nice?

**Comment by darksonic300** [comment]
Must say, I watched the live rn and it was pretty unlucky XD.
Virulent Quicksand was supposed to give suffocation damage only when players' head were inside of them, though for some wacky issues with block height I just had to give it damage to the touch.
Removing the damage altogether would render the quicksand in itself useless, so my thought goes not in removing, but debuffing both the duration of the inebriation effect on contact with poison, and less suffocation damage while in quicksand.
This gives the player more of a chance to survive while still presenting itself as a bit of a challenge, that's also a bit of the idea of the biome. Right now the Yagroot swamps don't have many reasons to be explored, but rest assured that will change too.
